lib/checks/navigation/region-evaluate.js

```diff
@@ -86,6 +91,14 @@ function findRegionlessElms(virtualNode, options) {
   }
 }
 
+function isShallowlyHidden(virtualNode) {
+  // The element itself is not visible to screen readers, but its descendants might be
+  return (
+    getRole(virtualNode, { noPresentational: true }) === null &&
+    !hasChildTextNodes(virtualNode)
```

Comment:
I don't think we can just ignore anything presentational. Most elements don't have this dynamic thing going on where they are only presentational if there is nothing interesting about them (alt, tabindex, global aria). We'd have to test for all of them if you want this to work for anything except the `img` element.

I think we're better off just skipping decorative images explicitly here, and leaving the rest for another day:

```suggestion
function isDecorativeImage(virtualNode) {
  return (
    virtualNode.props.nodeName === 'img' &&
    getRole(virtualNode, { noPresentational: true }) === null
```